test/func.js

```diff
@@ -521,4 +554,585 @@ describe('func', function () {
+    }
+    let filterItemCorrect2 = {
+      "data" : {
+        "trunk":{"$lte":5},
+      }
+    }
+    let filterItemIncorect = {
+      "data" : {
+        "trunk":{"$lt":5},
+      }
+    }
+    let filterItemIncorect2 = {
+      "data" : {
+        "trunk":{"$lte":4},
+      }
+    }
+
+    AddSearchDelete("/criteriaConversionsLt",dataItemAdded,filterItemCorrect,filterItemIncorect).then((data)=>{
+
+      return  AddSearchDelete("/criteriaConversionsLte",dataItemAdded,filterItemCorrect2,filterItemIncorect2)
+    }).then((data)=>{
+      done();
+    }).catch(done)
+
+
+  })
+
+  it('Criteria Conversion - Mongo Operator in  ',  function(done) {
+    let testIdNew = require('shortid').generate();
+
+    let dataItemAdded = {
+      "data" : {
+        "trunk":"hello",
+        "trunk2" : {
+          "branch1" : "branchLeaf",
+          "branch2" : {"twig" : "5" }
+        }
+      }
+    }
+    let filterItemCorrect = {
+      "data" : {
+        "trunk":{"$in":["goodbye","hello"]},
+      }
+    }
+    let filterItemCorrect2 = {
+      "data" : {
+        "trunk":"hello",
+        "trunk2" : {
+          "branch1" : {"$in":["branchLeaf","test"]},
+          "branch2" : {"twig" : "5" }
+        }
+      }
+    }
+
+    let filterItemIncorect = {
+      "data" : {
+        "trunk":{"$in":["Wrong"]},
+      }
+    }
+    let filterItemIncorect2 = {
+      "data" : {
+        "trunk":{"$in":["Wrong","VeryWorng","hell"]},
+      }
+    }
+
+    AddSearchDelete("/criteriaConversionsIn",dataItemAdded,filterItemCorrect,filterItemIncorect).then((data)=>{
+
+      return  AddSearchDelete("/criteriaConversionsIn",dataItemAdded,filterItemCorrect2,filterItemIncorect2)
+    }).then((data)=>{
+      done();
+    }).catch(done)
+  })
+
+  it('Criteria Conversion - Mongo Operator ne  ',  function(done) {
+    let testIdNew = require('shortid').generate();
+
+    let dataItemAdded = {
+      "data" : {
+        "trunk":"hello",
+        "trunk2" : {
+          "branch1" : "branchLeaf",
+          "branch2" : {"twig" : "5" }
+        }
+      }
+    }
+    let filterItemCorrect = {
+      "data" : {
+        "trunk":{"$ne":"goodbye"},
+      }
+    }
+    let filterItemCorrect2 = {
+      "data" : {
+        "trunk":"hello",
+        "trunk2" : {
+          "branch1" : {"$ne":"NotbranchLeaf"},
+          "branch2" : {"twig" : "5" }
+        }
+      }
+    }
+
+    let filterItemIncorect = {
+      "data" : {
+        "trunk":{"$ne":"hello"},
+      }
+    }
+    let filterItemIncorect2 = {
+      "data" : {
+        "trunk2" : {
+          "branch1" : {"$ne":"branchLeaf"},
+        }
+      }
+    }
+
+    AddSearchDelete("/criteriaConversionsNe",dataItemAdded,filterItemCorrect,filterItemIncorect).then((data)=>{
+
+      return  AddSearchDelete("/criteriaConversionsNe",dataItemAdded,filterItemCorrect2,filterItemIncorect2)
+    }).then((data)=>{
+      done();
+    }).catch(done)
+  })
+
+  it('Criteria Conversion - Mongo Operator in  ',  function(done) {
```

Comment:
Is this a duplicate test? There is another in test above

Reply:
duplicate removes